### ISSUE TITLE
Exploratory: redundant ".js" in names.

### ISSUE THREAD

**Issue by joshjordan** [tenex/rails-assets#338]
From @jnylen: fix the "honey.js.js" thing when the name itself have a ".js"

**Comment by joramwangi** [comment]
This also affects cleave.js

**Comment by hut8** [comment]
@joramwangi Do you mean that in the application.js file, you need to include cleave.js.js? Can you please explain the problem you're encountering?

**Comment by joramwangi** [comment]
Yes, you have to include it as "cleave.js.js"
Looking in the gem, that's the way the file is named: with an extra ".js"

**Comment by hovancik** [comment]
I am facing the same issue with `cleave.js`. In my `application.js` I had to use `cleave.js.js`

This should be either added to [install instructions](https://rails-assets.org/#/components/cleave.js) or fix internally.

**Comment by sheerun** [comment]
This is on purpose to not clash names with "cleave" package

**Comment by hovancik** [comment]
Yes, but [install instructions](https://rails-assets.org/#/components/cleave.js) are still wrong. 


![screen shot 2017-06-28 at 15 17 13](https://user-images.githubusercontent.com/4745859/27638898-f9d17330-5c14-11e7-888b-2617e9455718.png)

**Comment by fidalgo** [comment]
Same for `print.js`
